private/buf/bufctl/controller.go

```diff
@@ -1023,6 +1023,7 @@ func (c *controller) buildTargetImageWithConfigs(
 		c.logger.Debug(
 			"building image for target module",
 			zap.String("moduleOpaqueID", module.OpaqueID()),
+			zap.String("moduleDescription", module.Description()),
```

Comment:
Output:

```
buf lint foo --debug
...
DEBUG	building image for target module	{"moduleOpaqueID": "foo-00001", "moduleDescription": "path: \"foo\", excludes: \"foo/bar\""}
...
```

We could also just get rid of the opaque ID here, but I'd keep it for debugging purpose.